benchmark/benchmark.iml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this generated by IntelliJ? Should probably not include it in the commit?

Reply:
Woops, missed this. I'll fix

Reply:
ooops, missed too, sorry !